bulk-model-sync-mps/src/main/kotlin/org/modelix/mps/model/sync/bulk/IncludedModulesFilter.kt

```diff
@@ -19,23 +19,34 @@ package org.modelix.mps.model.sync.bulk
 import org.modelix.model.api.BuiltinLanguages
 import org.modelix.model.api.INode
 import org.modelix.model.sync.bulk.ModelSynchronizer
+import org.modelix.model.sync.bulk.isModuleIncluded
 
 /**
- * A filter that skips nodes, which represent MPS modules and do not match
- * the included module names ([includedModules]) or prefixes ([includedModulePrefixes]).
+ * A filter that skips nodes, which
+ *  (1) represent MPS modules and
+ *  (2) do not match the included module names ([includedModules]) or prefixes ([includedModulePrefixes]) and
+ *  (3) are not excluded via [excludedModules] or [excludedModulesPrefixes].
  *
  * Note: This is currently not meant to be used standalone.
  * It should be used with other filters in a [CompositeFilter].
  */
 class IncludedModulesFilter(
     val includedModules: Collection<String>,
     val includedModulePrefixes: Collection<String>,
+    val excludedModules: Collection<String> = emptySet(),
```

Comment:
The excludes were not passed to `IncludeModulesFilter`.

```kt
val repository = getRepository()

val includedModuleNames = parseRawPropertySet(System.getProperty("modelix.mps.model.sync.bulk.input.modules"))
val includedModulePrefixes = parseRawPropertySet(System.getProperty("modelix.mps.model.sync.bulk.input.modules.prefixes"))

val includedModulesFilter = IncludedModulesFilter(includedModuleNames, includedModulePrefixes)
```

Reply:
fixed